setup.py

```diff
@@ -27,13 +38,13 @@
 		'numpy==1.18.5',
 		'protobuf==3.17.3',
 		'pymongo==3.12.0',
-		'scikit-learn==0.24.2',
+		'scikit-learn',
 		'nibabel==3.2.1',
 		'resource==0.2.1',
 		'networkx==2.5.1',
 		'pandas==1.2.5',
 		'pathlib',
-                'DeepMRSeg @ git+https://github.com/CBICA/DeepMRSeg.git@main'
+        'DeepMRSeg @ git+https://github.com/CBICA/DeepMRSeg.git@main'
```

Comment:
This should be removed since we are going to nnunet version.

Reply:
Agree. I left it there in case somebody wanted to keep the old DeepMRSeg approach, but I realize that the code is commented out for the nipype interface.